libs/utils/utils.js

```diff
@@ -1026,6 +1026,7 @@ async function checkForPageMods() {
 }
 
 async function loadPostLCP(config) {
+  document.dispatchEvent(new Event(MILO_EVENTS.POSTLCP));
```

Comment:
can we move this at the bottom of loadPostLCP? i would give milo loadPostLCP calls priority over consumer's

Reply:
@npeltier makes sense - I can move this one yes 👍